main_test.go

```diff
@@ -740,46 +563,45 @@ OK
 }
 
 func TestLot(t *testing.T) {
-	t.Parallel()
+	// t.Parallel()
 	path := "_examples/lot"
 	testPkg(t, pkg{
 		path: path,
-		lang: []string{"py2", "py2-cffi", "py3-cffi", "pypy2-cffi", "pypy3-cffi"},
+		lang: features[path],
 		want: []byte(`l.SomeString : some string
 l.SomeInt : 1337
 l.SomeFloat : 1337.1337
 l.SomeBool : True
-l.SomeListOfStrings: []string{"some", "list", "of", "strings"}
-l.SomeListOfInts: []int64{6, 2, 9, 1}
-l.SomeListOfFloats: []float64{6.6, 2.2, 9.9, 1.1}
-l.SomeListOfBools: []bool{true, false, true, false}
+l.SomeListOfStrings: go.Slice_string len: 4 handle: 2 [some, list, of, strings]
```

Comment:
we should probably keep some quotes around the elements. either `"` as it used to or `'` (which may be more pythonic.)

Reply:
i'm not sure how to detect the slice element type in python wrapper properly, that works across versions etc.